src/lib/repositories_v2/impl/chef_repositoty_impl.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime
+
+from src.constants.audit import Status
+from src.core.ioc import get_ioc_instance
+from src.lib.entities.sqlalchemy_orm_mapping import Chef, Order
+from src.lib.repositories_v2.chef_repository import ChefRepository
+
+
+class ChefRepositoryImpl(ChefRepository):
+    def __init__(self):
+
+        ioc = get_ioc_instance()
+        self.session = ioc.get_instance("orm_session")
+
+    def add(self, chef):
+        chef.created_date = datetime.now()
+        chef.updated_by = chef.created_by
+        chef.updated_date = chef.created_date
+        self.session.add(chef)
+        self.session.commit()
+
+    def get_by_id(self, chef_id):
+        return (
+            self.session.query(Chef)
+            .filter(Chef.id == chef_id)
+            .filter(Chef.entity_status == "ACTIVE")
+            .first()
+        )
+
+    def get_all(self):
+        chefs = self.session.query(Chef).filter(Chef.entity_status == "ACTIVE")
+        return chefs
+
+    def delete_by_id(self, chef_id, chef):
+        chef_to_be_delete = self.session.query(Chef).filter(Chef.id == chef_id).first()
```

Comment:
I think you just need the id here. Why don't you use an update query? I don't think you need to load the entity at all.

Reply:
Well yes, maybe you need the chef to get the updated by, which makes sense.

Reply:
Well yes, maybe you need the chef to get the updated by, which makes sense.

Reply:
yes, it is.